xsl/xccdf-report-impl.xsl

```diff
@@ -852,6 +852,11 @@ Authors:
                         </xsl:apply-templates>
                 </div></td></tr>
             </xsl:if>
+            <xsl:if test="$profile/cdf:select[@idref=$item/@id]/cdf:remark/text()">
```

Comment:
Just an FYI, I was reading the standard tonight:

> The `<xccdf:select>` element MAY have one or more `<xccdf:remark>`
> elements (strings) containing explanatory material or other prose. Each
> instance of `<xccdf:remark>` MAY have an `@xml:lang` attribute (see
> Section 6.2.10).

IIUC, this doesn't handle having multiple remarks under the selected item. You'll probably need to iterate over each remark and add it to a separate section.

Reply:
This should fixed in my latest commit.